app/assets/stylesheets/components/_buttons.scss

```diff
@@ -0,0 +1,17 @@
+.btn-gold {
+  background-color: transparent;
+  padding: 10px 20px;
+  text-decoration: none;
+  font-size: 16px;
+  border-radius: 4px;
+  display: inline-block;
+  text-align: center;
+  color: $gold;
+  transition: .3s;
+  border: 1px solid $gold;
+}
+
+.btn-gold:hover {
+  background-color: $gold;
+  color: $main-grey;
+}
```

Comment:
In scss you can indent elements inside another.
Can you change that like this please ?
```
.btn-gold {
  ...
  ...
  
  &:hover {
    ...
    ...
    }
  }
```